packages/bot/src/stores/run-panel-store.js

```diff
@@ -56,7 +57,8 @@ export default class RunPanelStore {
 
     @action.bound
     onRunButtonClick = () => {
-        const { client } = this.root_store.core;
+        const { core : { client }, contract_card } = this.root_store;
```

Comment:
Imo it's clearer to only destructure one level.

```suggestion
    const { client }        = this.root_store.core;
    const { contract_card } = this.root_store;
```